### ISSUE TITLE
[FEATURE] Add alt text for user risk

### ISSUE THREAD

**Issue by f-bader** [glueckkanja/MyWorkID#55]
If the user hovers over the risk state text there should be an alt text with additional information

* None - No current indications of risk.
* Low - Minimal risk, no immediate action required.
* Medium - Potential risk, monitoring recommended.
* High - Severe risk, urgent intervention needed.

[Source](https://teams.microsoft.com/l/message/19:24077fd1f4304e00a35fba14b5812b2d@thread.tacv2/1737969033221?tenantId=a53834b7-42bc-46a3-b004-369735c3acf9&groupId=1369af43-d581-46d6-b1fd-a2d32b56a9f3&parentMessageId=1737111265342&teamName=GK%20Tenant&channelName=General&createdTime=1737969033221)
